src/main/java/frc/robot/FieldVisionInputGenerator.java

```diff
@@ -0,0 +1,22 @@
+package frc.robot;
```

Comment:
move all your classes for this purpose into a different package, not under frc.robot. we discussed this